resolwe_bio/docker_images/chipseq/README.md

```diff
@@ -1,13 +1,14 @@
 # Docker image for ChIP-Seq processes
 
-It is based on `ubuntu-14.04` version of [`docker.io/resolwe/base`](
+It is based on `ubuntu-17.04` version of [`docker.io/resolwe/base`](
 https://hub.docker.com/r/resolwe/base/) image.
 
 Included bioinformatics tools
 -----------------------------
 * ROSE2 (v1.0.5)
 * BAMPLOT (v0.9.0)
-* MACS (v1.4.2)
+* MACS (v1.4.3)
 * MACS2 (v2.1.1.20160309)
 * BAMLIQUIDATOR (v1.3)
 * SAMTOOLS (v0.1.19)
```

Comment:
samtools version is probably not correct anymore

Reply:
samtools version is the same